pkg/server/metrics.go

```diff
@@ -68,6 +71,13 @@ var _initMetricsFunc = sync.OnceValue[*metrics](func() *metrics {
 		Help: "The total number of new dsse log entries",
 	})
 
+	// grpc_packet_part should always be "payload" but we can measure "header" or "trailer" in the future
+	// if we so desire
+	m.grpcRequestSize = f.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "grpc_api_request_size",
```

Comment:
name should start with rekor for consistency

Reply:
this is actually consistent with the stats produced by grpc's default server metrics set.
see: https://github.com/grpc-ecosystem/go-grpc-middleware/blob/main/providers/prometheus/server_metrics.go#L33